AppCreationScripts/apps.json

```diff
@@ -28,15 +28,20 @@
           ]
         }
       ],
-      "codeConfigurations": [	
+      "codeConfigurations": [
         {
-		      "settingFile": "/active-directory-wpf-msgraph-v2/App.xaml.cs", 
-		      "replaceTokens": 
-            {
-              "appId": "4a1aa1d5-c567-49d0-ad0b-cd957a47f842",
-              "tenantId": "common"
-			      }
-         }
+          "settingFile": "/active-directory-wpf-msgraph-v2/App.xaml.cs",
+          "replaceTokens": {
+            "appId": "4a1aa1d5-c567-49d0-ad0b-cd957a47f842",
+            "tenantId": "common",
+            "authorityEndpointHost": "https://login.microsoftonline.com/"
+          }
+        },
+        {
+          "settingFile": "/active-directory-wpf-msgraph-v2/MainWindow.xaml.cs",
+          "replaceTokens": {
+            "msgraphEndpointHost": "https://graph.microsoft.com"
```

Comment:
Is the portal going to replace with or without the trailing slash @archieag  ?

Reply:
@jmprieur The portal will replace it with a trailing slash.

Reply:
@aiwangmicrosoft : then we want to have   "msgraphEndpointHost": "https://graph.microsoft.com/"

Reply:
The latest commit added the trailing slash.